src/python/zquantum/qubo/conversions.py

```diff
@@ -33,8 +39,13 @@ def convert_qubo_to_openfermion_ising(qubo: BinaryQuadraticModel) -> IsingOperat
 
 def convert_openfermion_ising_to_qubo(operator: IsingOperator) -> BinaryQuadraticModel:
     """
-    Converts dimod BinaryQuadraticModel to OpenFermion IsingOperator object.
-    NOTE: The conversion might not be 100% accurate due to performing floating point operations during conversion between Ising and QUBO models.
+    Converts dimod Openfermion IsingOperator to BinaryQuadraticModel object.
+    The resulting QUBO has the following property:
+    For every bitstring, its expected value is the same as the one of the original Ising Hamiltonian.
```

Comment:
Same as above: "For every bitstring, its energy is the same as the one of the original Ising Hamiltonian.